src/software/ai/hl/stp/play_info_test.cpp

```diff
@@ -0,0 +1,86 @@
+#include "software/ai/hl/stp/play_info.h"
+
+#include <gtest/gtest.h>
+
+#include <string>
+
+TEST(PlayInfoTest, get_play_type_works)
+{
+    std::string s, empty;
```

Comment:
nit: Can we keep the variable delcaration and initialization on the same line? So this would become
```
std::string s = "new type";
std::string empty = "";
```
This is just generally easier to read for people.

Also can we avoid using single-letter variable names like `s`? People may not know if that means "string", or is supposed to stand for something else. Longer but more descriptive names are always preferred. You can check out our style guide here: https://github.com/UBC-Thunderbots/Software/blob/master/docs/code-style-guide.md

This will need to changed throughout this file, just gonna leave a single comment here rather than comment everywhere.

Reply:
I was actually trying to declare and initialize on the same line, but CLion gives me this error:
<img width="1440" alt="Screen Shot 2019-11-21 at 6 04 10 PM" src="https://user-images.githubusercontent.com/45518450/69392023-75862580-0c89-11ea-9894-5a73fcf5cbfd.png">

And I am not sure how I am supposed to resolve it.  I remember reading somewhere in the docs that I shouldn't cast the variable...

Reply:
The code should compile despite that warning, but what it's basically saying is that the string variable is never changed after it is assigned so should be declared as `const` (so the compiler can help verify the value never changes).

Rather than casting you should be able to just prepend the `const` keyword in this case.

Eg. `const std::string my_variable_name = "foobar";`